terraform/scripts/mongodb.tpl

```diff
@@ -14,3 +14,4 @@ db.rules.insert({name:"warning_file_types", source:"print('$${file.Filename}'['$
 db.rules.insert({name:"restricted_file_types", source:"print('$${file.Filename}'['$${file.Filename}'.rfind('.')+1:] not in ['zip','tgz','gzip','tar','com','exe','dll','scr','sas7bdat','spv','rda','dta'])",mandatory:true});
 db.rules.insert({name:"file_size_under_3.5mb", source:"print($${file.size}<3670016)",mandatory:false });
 db.rules.insert({name:"file_size_under_5mb", source:"print($${file.size}<5242880)",mandatory:true });
+db.rules.insert({name:"studyid_in_content", source:"print(not bool(re.search(r'[\\w]{1}[\\d]{9}', $${file.content}.decode('utf-8'))))", mandatory: true});
```

Comment:
Add a comment describing what this rule does for clarity, since it's js (mongo) you can use //